scalability_and_performance/low_latency_tuning/cnf-performing-platform-verification-latency-tests.adoc

```diff
@@ -15,11 +15,17 @@ The `cnf-tests` container image is available at `registry.redhat.io/openshift4/c
 
 Your cluster must meet the following requirements before you can run the latency tests:
 
-. You have configured a performance profile with the Node Tuning Operator.
+* You have installed the real-time kernel in the cluster.
```

Comment:
Real time kernel is not necessary, it will influence the results though. Telco Core is not using is for example.

Reply:
Thanks @MarSik for your feedback. I have removed real-time kernel as a prerequisite.